alf/utils/common.py

```diff
@@ -854,18 +881,13 @@ def decorate(f):
 def set_random_seed(seed):
     """Set a seed for deterministic behaviors.
 
-    Note: If someone runs an experiment with a pre-selected manual seed, he can
-    definitely reproduce the results with the same seed; however, if he runs the
-    experiment with seed=None and re-run the experiments using the seed previously
-    returned from this function (e.g. the returned seed might be logged to
-    Tensorboard), and if cudnn is used in the code, then there is no guarantee
-    that the results will be reproduced with the recovered seed.
-
```

Comment:
should not remove

Reply:
Seems to be caused by a merge. Changed back.

Reply:
There are a number of unintended changes in ```common.py``` due to a merge with PR_batch_env_temp for testing. They are changed back.